### DISCUSSION TITLE
Getting custom fields thought GraphQL

### DISCUSSION THREAD

**Discussion by curtishenson** [verbb/hyper#68] [Q&A] [answered]
Is it possible to get custom fields in a link through graphql? Not sure if I'm just doing it wrong or there isn't a way?

Currently trying to get an asset field, haven't tried other field types yet.

**Comment by engram-design** [comment]
Not at the moment, sorry. I'll need to see how I can make this a non-breaking change, without forcing everyone to change their GQL queries to include the link type (to gain access to the custom fields).

**Reply by engram-design** [reply, marked as answer]
@curtishenson Added for the next release. To get this early run `composer require verbb/hyper:"dev-craft-4 as 1.1.7"`

**Reply by curtishenson** [reply]
Awesome thank you!

**Comment by engram-design** [comment]
Updated in [3.0.0-beta.1](https://github.com/verbb/hyper/releases/tag/3.0.0-beta.1).
